libraries/aws/shadow/src/aws_iot_shadow_api.c

```diff
@@ -108,6 +105,8 @@ static AwsIotShadowError_t _setCallbackCommon( IotMqttConnection_t mqttConnectio
                                                _shadowCallbackType_t type,
                                                const char * pThingName,
                                                size_t thingNameLength,
+											   const char * pShadowName,
```

Comment:
Spacing off on these?

Reply:
Maybe I dont want to run amazons code through astyle as it will make future merging a major pain.